services/wallet/responses/router_suggested_routes.go

```diff
@@ -6,10 +6,9 @@ import (
 )
 
 type RouterSuggestedRoutes struct {
-	Uuid                  string                `json:"Uuid"`
-	Best                  routes.Route          `json:"Best,omitempty"`
-	Candidates            routes.Route          `json:"Candidates,omitempty"`
-	TokenPrice            *float64              `json:"TokenPrice,omitempty"`
```

Comment:
@Khushboo-dev-cpp sorry, I missed the following... since you've replaced `TokenPrice` and  `NativeChainTokenPrice` with `UpdatedPrices` map, these changes are breaking changes, so please:
- update your commit message to `feat!: prices map returned along with SuggestedRoutes instead native token and token prices`
- add that under section **Breaking change!** to the PR's description